prodigy_demo/README.md

```diff
@@ -0,0 +1,149 @@
+## 🪄 Prodigy have-a-go session
+
+In this have-a-go session we will explore **Prodigy**, a data annotation tool by [Explosion AI](https://explosion.ai/) (the creators of [spaCy](https://spacy.io/)). Prodigy allows you to flexibly annotate data: text, images, video and audio.
+
+In order to use Prodigy for your annotation tasks you need a license. We have 5 seats in the Data Science Practice, which are freely transferrable within the company. However, if you do need a license for a project, please join Nesta's `#dsp-prodigy` slack channel and ask for a license key.
+
+### 🛠️ Setup
+
+Run the following on your command line before the have-a-go session:
+1. If you don't have this repo in your local computer run: `git clone https://github.com/nestauk/dap_tutorials.git`; Otherwise open this repo locally and run `git fetch origin`;
+2. `cd YOUR_LOCAL_PATH/dap_tutorials/prodigy_demo/` where `YOUR_LOCAL_PATH` is your local path to your repos' folder;
+2. `conda create --name prodigy_demo python=3.9`
+3. `conda activate prodigy_demo`
+4. `pip install -r requirements.txt`
+5. `pip install -r ./llm_recipe/llm_requirements.txt` - specific requirements for the llm recipe;
+6. `echo OPENAI_KEY=<YOUR_API_KEY> >> .env` where `YOUR_API_KEY` is your OpenAI API key (we will make one available for you);
```

Comment:
```suggestion
6. `echo OPENAI_KEY="YOUR_API_KEY" >> .env` where `YOUR_API_KEY` is your OpenAI API key (we will make one available for you);
```

Reply:
@india-kerle not sure about this one but I think you need to use "" instead of <>, but perhaps both work?